src/redux/missions/missions.js

```diff
@@ -1,6 +1,8 @@
 const initialState = [];
 
 const GET_DATA_MISSIONS = 'GET_DATA_MISSIONS';
+const JOIN_MISSION = 'JOIN_MISSION';
+const LEAVE_MISSION = 'LEAVE_MISSION';
```

Comment:
[optional] A good practice could be if you implement the duck's pattern [See Here](https://github.com/erikras/ducks-modular-redux), for example your `'GET_DATA_MISSIONS'` action can be formatted like `space-travelers-hub/missions/GET_DATA_MISSIONS`.